tilt-schema.json

```diff
@@ -746,12 +730,15 @@
               },
               "category": {
                 "$id": "#/properties/dataDisclosed/items/anyOf/0/properties/category",
-                "type": "string",
+                "type": "array",
                 "title": "Category",
-                "description": "The data (category) the data disclosed is referred to.",
+                "description": "The data (category) the data disclosed are referred to.",
```

Comment:
In die description vielleicht noch sowas wie "can be multiple".

Reply:
done